code/modules/religion/rites.dm

```diff
@@ -109,6 +109,29 @@
 	human2borg.visible_message("<span class='notice'>[human2borg] has been converted by the rite of [name]!</span>")
 	return TRUE
 
+/datum/religion_rites/upgrade_blessing
+	name = "Receive Blessing"
+	desc = "Receive a blessing from the machine gods to further your ascension"
+	ritual_length = 15 SECONDS
+	ritual_invocations = list("With hammer and faith our bodies were forged",
+							"Oh, Machine-God, We are your instruments of faith",
+							"Let your energy overload our iron-forged blows")
+	invoke_msg = "The end of flesh is near!"
+	favor_cost = 2500
+
+datum/religion_rites/upgrade_blessing/invoke_effect(mob/living/user, atom/movable/religious_tool)
```

Comment:
LOOK AT THIS MAN, HE RELATIVELY PATHED HIS PROC

Reply:
GET OUT OF MY HEADGET OUT OF MY HEADGET OUT OF MY HEADGET OUT OF MY HEADGET OUT OF MY HEADGET OUT OF MY HEADGET OUT OF MY HEADGET OUT OF MY HEAD